mpas_analysis/ocean/time_series_ocean_regions.py

```diff
@@ -728,9 +730,15 @@ def run_task(self):  # {{{
 
         for var in self.variables:
             varName = var['name']
-            title = 'Volume-Mean {} in {}'.format(var['title'],
-                                                  self.regionName)
             mainArray = dsIn[varName]
+            is3d = mainArray.attrs['is3d'] == 'True'
```

Comment:
this confuses me a bit: isn't it always attributing the value `True` to `is3d` no matter what `mainArray.attrs['is3d']` is?

Reply:
Nope, the expression `mainArray.attrs['is3d'] == 'True'` evaluates to `True` if the attribute `is3d` is equal to the string `'True'` and `False` otherwise.

Reply:
Maybe the difference between the single equals and the double equals is what makes this confusing?

Reply:
It could be changed to `is3d = (mainArray.attrs['is3d'] == 'True')` if that would be clearer for you.

Reply:
no, that's fine, it's just pythonic thinking that I still don't have :) Basically the expression with the double equal is an if statement, now I see it.